src/unix/uclibc/mod.rs

```diff
@@ -2318,6 +2319,15 @@ extern "C" {
     pub fn getgrgid(gid: ::gid_t) -> *mut ::group;
     pub fn popen(command: *const c_char, mode: *const c_char) -> *mut ::FILE;
     pub fn uname(buf: *mut ::utsname) -> ::c_int;
+    pub fn getnameinfo(
+        sa: *const ::sockaddr,
+        salen: ::socklen_t,
+        host: *mut ::c_char,
+        hostlen: ::socklen_t,
+        serv: *mut ::c_char,
+        sevlen: ::socklen_t,
+        flags: ::c_int,
```

Comment:
```suggestion
        flags: ::c_uint,
```

Should be?

Reply:
emmm..It should be `uint` on uclibc.
https://docs.rs/dns-lookup/1.0.5/dns_lookup/fn.getnameinfo.html
It seems that conditional compilation is needed here.